internal/services/servicebus/servicebus_queue_resource.go

```diff
@@ -295,6 +325,24 @@ func resourceServiceBusQueueCreateUpdate(d *pluginsdk.ResourceData, meta interfa
 		return err
 	}
 
+	log.Printf("[DEBUG] Waiting for %s status to become ready", id)
```

Comment:
presumably the CreateOrUpdate is now returning a Future?

Reply:
unfortunately, the api is not returning the future, we still need to wait the status to be ready

Reply:
then this sounds like a bug on the API side, could we please open an issue on the rest API specs detailing what is happening and the workaround we are doing to make it work then link it here for future refrences. once thats doine this will be good to merge